src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.scss

```diff
@@ -1 +1,52 @@
+.tooltip {
+  display: flex;
+  flex-direction: row;
+  border-radius: 5px;
+  background: var(--rundown-scrollbar-color);
+  height: 110px;
```

Comment:
I've removed the set height on content, but kept it on the hover scrub itself. It's due to the fact that if no preview is available, the video will get so large that the tooltip will become way to huge, and it will also flicker before the preview is set.